src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx

```diff
@@ -30,16 +30,12 @@ const MediaPlayerInnerWrapper = styled.div`
 
 const LiveRadioContainer = ({ idAttr, externalId, id }) => {
   const { platform, origin } = useContext(RequestContext);
-  const { liveRadio, lang, translations } = useContext(ServiceContext);
+  const { liveRadio, lang, translations, service } = useContext(ServiceContext);
 
   const isAmp = platform === 'amp';
 
-  const MediaPlayer = {
-    canonical: CanonicalMediaPlayer,
-    amp: AmpMediaPlayer,
-  }[platform];
-
-  if (!MediaPlayer || !externalId || !id) return null;
+  if (!['amp', 'canonical'].includes(platform) || !externalId || !id)
```

Comment:
could this `!['amp', 'canonical'].includes(platform)` be `invalidPlatform` so it's easier to read?

Reply:
Great suggestion. Amended.